resources/lib/dplay.py

```diff
@@ -540,21 +541,23 @@ def get_stream(self, video_id, video_type):
 
         params = {'usePreAuth': 'true'}
 
+        jsonPayload = {'deviceInfo': {'adBlocker': 'true'}, 'videoId': video_id, 'wisteriaProperties':{'product':'dplus_us'}}
+
         # discoveryplus.com (US)
         if self.locale_suffix == 'us':
-            url = '{api_url}/playback/v3/videoPlaybackInfo/{video_id}'.format(api_url=self.api_url, video_id=video_id)
+            url = '{api_url}/playback/v3/videoPlaybackInfo'.format(api_url=self.api_url, video_id=video_id)
         else:
             if video_type == 'channel':
                 url = '{api_url}/playback/v2/channelPlaybackInfo/{video_id}'.format(api_url=self.api_url, video_id=video_id)
             else:
                 url = '{api_url}/playback/v2/videoPlaybackInfo/{video_id}'.format(api_url=self.api_url, video_id=video_id)
 
-        data_dict = json.loads(self.make_request(url, 'get', params=params, headers=self.site_headers))['data']
+        data_dict = json.loads(self.make_request(url, 'post', params=params, headers=self.site_headers, jsonPayload=jsonPayload))['data']
```

Comment:
Is parameter jsonPayload actually needed? Could this be payload=json.dumps(jsonPayload) ? Like here https://github.com/Dis90/plugin.video.discoveryplus/blob/ae08554d437f873419a3554ecfb7a02a6c990225/resources/lib/dplay.py#L268

Reply:
changed it like your example and it works